src/DeleteButton.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import  Button  from 'react-bootstrap/Button'
+
+
+
+
+export default class DeleteButton extends Component {
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+  
+   
+    this.props.deleteBook();
+  }
+
+  render() {
+    return (
+      <div>
+         <Button type ="submit">Delete</Button>
+
```

Comment:
- backdent here, no whitespace needed, but `<div>` indententation can be fixed 👍